components/Internship/CommitteeInternshipInfo.js

```diff
@@ -0,0 +1,67 @@
+import Image from 'next/image';
+import React, {useState} from 'react';
+
+function CommitteeInternshipInfo(props) {
```

Comment:
One thought on this function - if we'd like, we can create a `board` committee entry (i.e. an object with the relevant keys), and then just prepend/append it to the props; this can let us reduce duplication, since the implementation itself can rely on just interpreting `committees` instead of creating a special case for board.

Reply:
Hm I tried doing this, but there seems to be a problem with `useEffect` in that it doesn't run automatically when the page loads but only once I click on one of the other logos - will push these changes in a bit!

Reply:
(since it isn't running on page load, it isn't showing up in the next steps cards either)

Reply:
Why would you need to use `useEffect` here? I mean running this command on build, i.e. creating a new object at the top-level and prepending.

Reply:
directly doing that for some reason duplicates the `board` entry, adding it twice (and sometimes more times) - I'm not sure why it's happening but I think it's something to do with it prepending again on each render - which is why I switched to `useEffect`?

Reply:
So, you'd run this top-level (i.e. not in the render function of the component) - that way, the expression gets run once when the variable is first evaluated, and that would be it. That way, you shouldn't need `useEffect`, and the expression gets run and built at build time.